HoloLensCommander/HoloLensCommander/MainWindowViewModel.cs

```diff
@@ -299,7 +441,16 @@ private void RegisterCommands()
             this.UninstallAllAppsCommand = new Command(
                 (parameter) =>
                 {
-                    this.UninstallAllApps();
+                    try
+                    {
+                        this.UninstallApp();
```

Comment:
THANK YOU! This looks to have been caused by a recent merge in my fork, The correct API is now being called